includes/controls/groups/typography.php

```diff
@@ -289,6 +289,7 @@ protected function get_default_options() {
 				'starter_title' => _x( 'Typography', 'Typography Control', 'elementor' ),
 				'settings' => [
 					'render_type' => 'ui',
+					'global' => true,
```

Comment:
?

Reply:
This is temporary. It is for the typography control JS, to not add the Global Control Select behavior in the Global Settings panel page.
We will need to handle this when we move the Global Control Select behavior to attach itself to the global controls with the filter in the behaviors() method.